modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/upgrade/v10_0_0/test/ObjectDefinitionUpgradeProcessTest.java

```diff
@@ -60,6 +59,25 @@ public void testUpgrade() throws Exception {
 
 		_assertObjectDefinitionPKObjectFieldPrefix(objectDefinition2, "c_");
 
+		String pkObjectFieldDBColumnName = StringUtil.randomId();
+		String pkObjectFieldName = StringUtil.randomId();
+
+		ObjectDefinition objectDefinition3 =
+			_addModifiableSystemObjectDefinition();
+
+		objectDefinition3 =
+			_objectDefinitionLocalService.publishSystemObjectDefinition(
+				TestPropsValues.getUserId(),
+				objectDefinition3.getObjectDefinitionId());
+
+		objectDefinition3.setPKObjectFieldDBColumnName(
+			pkObjectFieldDBColumnName);
+		objectDefinition3.setPKObjectFieldName(pkObjectFieldName);
+
+		objectDefinition3 =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				objectDefinition3);
```

Comment:
You could replace this code with `ObjectDefinitionTestUtil.addModifiableSystemObjectDefinition()` + `publishSystemObjectDefinition()`

Reply:
Done!